cmd/talosctl/cmd/talos/track.go

```diff
@@ -18,6 +18,6 @@ type trackableActionCmdFlags struct {
 
 func (f *trackableActionCmdFlags) addTrackActionFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&f.wait, "wait", true, "wait for the operation to complete, tracking its progress. always set to true when --debug is set")
-	cmd.Flags().BoolVar(&f.debug, "debug", false, "debug operation from kernel logs. --no-wait is set to false when this flag is set")
+	cmd.Flags().BoolVar(&f.debug, "debug", false, "debug operation from kernel logs. --wait is set to true when this flag is set")
```

Comment:
I believe `wait` is true by default since talos 1.2, right @utkuozdemir ?

Reply:
yes, it defaults to `--wait`, but still `--debug=true --wait=false` will imply `--wait=true`

Reply:
⬆️ yep. And `--no-wait` does not exist. So the change on the code is correct.